src/components v2/Atoms/Components/Figure/Figure.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="photon-figure" :class="[size]">
+    <span class="figure-container">
+      <div
+        class="figure"
+        :class="[$attrs.class, (!loading && 'animate-fade-in-slow') || '']"
+      >
+        {{ loading && isCurrency ? calculation : display }}
+      </div>
+    </span>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import { addCommaSeparators } from '@/utils';
+import { FiguresData } from './types';
```

Comment:
There's a ref to types though the file doesn't exist

Reply:
It should point towards the molecules types.